lib/mastodon/cli/base.rb

```diff
@@ -4,6 +4,7 @@
 require_relative '../../../config/environment'
 
 require 'thor'
+require 'pastel'
```

Comment:
What's the reason for this addition?

Reply:
Some of the warning/error lines emitted by the CLI::Media class use the `pastel` method from base to output things ... but those attempts were throwing missing constant errors because it wasn't available. I think this is a side effect of the pastel gem only being in the bundle because it's a dependency of tty-prompt, but tty-prompt has `require: false` in the bundle, so neither are loaded automatically.

Totally separate from this issue - I've noticed while adding some of this coverage that we still have some mixed use of thor/tty/pastel/etc across these classes. Will review this at some point and attempt another standardization after more of the coverage is in.